server/services/index.ts

```diff
@@ -6,24 +6,27 @@ import ImprisonmentStatusesService from './imprisonmentStatusesService'
 import TransfersService from './transfersService'
 import PrisonService from './prisonService'
 
-const { hmppsAuthClient, welcomeClientBuilder } = dataAccess
-const userService = new UserService(hmppsAuthClient, welcomeClientBuilder)
-const expectedArrivalsService = new ExpectedArrivalsService(hmppsAuthClient, welcomeClientBuilder)
-const temporaryAbsencesService = new TemporaryAbsencesService(hmppsAuthClient, welcomeClientBuilder)
-const imprisonmentStatusesService = new ImprisonmentStatusesService(hmppsAuthClient, welcomeClientBuilder)
-const transfersService = new TransfersService(hmppsAuthClient, welcomeClientBuilder)
-const prisonService = new PrisonService(hmppsAuthClient, welcomeClientBuilder)
+export const services = () => {
+  const { hmppsAuthClient, welcomeClientBuilder } = dataAccess()
```

Comment:
only call `dataAccess` to create connection if we are calling the `services` function